package-lock.json

```diff
@@ -1440,7 +1440,6 @@
       "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
```

Comment:
Looks like you maybe used the wrong version of npm to install these modules. You shouldn't have this big of a diff.